.github/workflows/test.yml

```diff
@@ -14,8 +14,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
```

Comment:
You should also upgrade to `checkout@v2` ;)

https://github.com/actions/checkout/releases/tag/v2.0.0

Reply:
<img width="74" alt="pr welcome" src="https://user-images.githubusercontent.com/1402241/74517412-06992f80-4f4d-11ea-9028-0cf1d33eb509.png">